### PR TITLE
chore: merge gaia 7.0.2 release

### PR DESCRIPTION
**Pull request by JimLarson**
Closes: Agoric/agoric-sdk#5773

For review, please pay attention to the following which received manual edits:

```
	.github/workflows/lint.yml
	CHANGELOG.md
	Makefile
	README.md
	app/ante_handler.go
	app/app.go
	app/migrate.go
	cmd/ag0/main.go
	cmd/gaiad/cmd/root.go
	cmd/gaiad/main.go
	docs/package.json
	go.mod
	go.sum
```

Still to do for upgrade: upgrade handler (?), test instructions, upgrade instructions, release notes

### CODE REVIEW

File: app/const.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package gaia
+
+// nolint: gosec
+const (
+	appName     = "GaiaApp"
+	upgradeName = "v7-Theta"
```
Comment (arirubinstein) [inline app/const.go]: i think we can just cheat and call this agoric-upgrade-7

Comment (JimLarson) [reply app/const.go]: And how about `appName`?

Comment (arirubinstein) [reply app/const.go]: should stay the same - at least it's that way in agoric-upgrade-5/6

**Comment by zmanian** [comment]
The liquidity module and gravity dex code should be removed from go.mod

`github.com/gravity-devs/liquidity v1.4.0`

**Comment by JimLarson** [comment]
The liquidity module is already removed from go.mod.

File: app/const.go (hunk 1 above)
Comment (JimLarson) [reply app/const.go]: Done.

**Comment by JimLarson** [comment]
Ready for approval and merge. Reminder that this isn't necessarily everything required for update-7.

**Review by arirubinstein — APPROVED** [review]

**Pull request merged**